helpers/markdownDocumentsReader.ts

```diff
@@ -17,12 +19,21 @@ export function getPostDocumentBySlug(slug: string) {
 
   return JSONSerialize({ slug: realSlug, ...data, content }) as PostDocument;
 }
-
 export function getAllPostDocuments(): PostDocumentWithoutContent[] {
   const slugs = fs.readdirSync(documentsDirectory);
   const docs = slugs
     .map((slug) => getPostDocumentBySlug(slug))
-    .filter((post: PostDocument) => post.draft === false)
+    .filter((post: PostDocument) => {
+      {
+        if (process.env.NODE_ENV === "development") {
+          return post;
```

Comment:
```suggestion
          return true;
```
It's a `filter` not `map`